backends/gdx-backend-robovm/src/com/badlogic/gdx/backends/iosrobovm/IOSUIViewController.java

```diff
@@ -79,7 +80,11 @@ public void viewDidLayoutSubviews () {
 		if (graphics.created && (newBounds.width != oldBounds.width || newBounds.height != oldBounds.height)) {
 			graphics.makeCurrent();
 			graphics.updateSafeInsets();
-			app.listener.resize(newBounds.width, newBounds.height);
+			if (graphics.config.hdpiMode == HdpiMode.Pixels) {
+				app.listener.resize(newBounds.width, newBounds.height);
+			} else {
+				app.listener.resize(newBounds.backBufferWidth, newBounds.backBufferHeight);
+			}
```

Comment:
Shouldn't it be the opposite?

Reply:
🤔 You're right, it probably should. Not sure how this worked...